docs/reference/numerical-types.md

```diff
@@ -104,18 +104,19 @@ printf("Index %" PRI_PlatformIndexType ". Min %" PRI_PlatformIndexType, index, s
 ```
 
 > [!NOTE]
-> in order for F´ to compile without warnings it is necessary that each of the platform types are elements in the set of integers supplied by the C standard integers header (`stdint.h`). i.e. each type must be an `int8_t`, `int16_t`, `int32_t`, `int64_t` or unsigned variants. On some compilers `int` and `unsigned int` are not members of that set and on those platforms it is imperative that both `PlatformIntType` and `PlatformUIntType` be set to some fixed size type instead.
+> in order for F´ to compile without warnings it is necessary that each of the platform types are elements in the set of integers supplied by the C standard integers header (`stdint.h`). i.e. each type must be an `int8_t`, `int16_t`, `int32_t`, `int64_t` or unsigned variants. On some compilers `int` and `unsigned int` are not members of that set and on those platforms `PlatformIntType` and `PlatformUIntType` should not be serialized.
```

Comment:
It is unclear to me what the following entails:
> on those platforms `PlatformIntType` and `PlatformUIntType` should not be serialized.

Does the user need to be careful about their usage, or will the framework help?